Makefile

```diff
@@ -139,8 +139,11 @@ docker-otelcontribcol:
 
 .PHONY: otelcontribcol
 otelcontribcol:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_$(GOOS)_$(GOARCH)$(EXTENSION) $(BUILD_INFO) ./cmd/otelcontribcol
+	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_$(GOOS)_$(GOARCH)$(EXTENSION) \
+		$(BUILD_INFO) ./cmd/otelcontribcol
```

Comment:
Would be nice to have separate build targets so build time doesn't increase.

Reply:
Make target, or really a new CircleCI build step? A new `make target` makes sense, but a new build step would probably take even longer, as it would need a runner to be started + the time go takes to download the dependencies, wouldn't it?

Reply:
I split the make target. Won't change CircleCI for now.
More changes are coming anyway, we need to have a way to run E2E tests against unstable executable.